package.json

```diff
@@ -33,7 +33,10 @@
   },
   "homepage": "https://github.com/xenova/transformers.js#readme",
   "dependencies": {
+    "image-decode": "^1.2.2",
+    "image-encode": "^1.3.1",
     "onnxruntime-web": "^1.14.0",
+    "resize-image-data": "^0.3.1",
```

Comment:
If possible, I'd like to not include dependencies like these.

`resize-image-data` only has ~1k weekly downloads and has canvas as a dependency. I've specifically avoided that since it has issues with installation in many environments (like electron). I've opted for [sharp](https://www.npmjs.com/package/sharp) instead.

Reply:
That said, sharp doesn't seem to be supported for [react native](https://github.com/lovell/sharp/issues/2229), but hopefully there's another way.

Reply:
For now, you could use the `interpolate_data` method I've defined in math_utils.js

Reply:
Ok.
I had just test this part.
And facing this problem. (This package use the `@canvas/image-data`, seems not work perfectly on RN